### PR TITLE
Change of unitest, setup, update requirement

### PR DESCRIPTION
**Pull request by Neilstid**

### CODE REVIEW

File: etc/setup.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# install
+mkdir ~/etc/
```
Comment (yuyiguo) [inline etc/setup.sh:3]: check if  ~/etc exists already.

File: etc/setup.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+FILE=~/etc/var.sh
+if [ ! -f $FILE ];
+then
+    >$FILE
+    echo "PythonFolder=$(dirname "$(which python)")" >> $FILE
```
Comment (yuyiguo) [inline etc/setup.sh:8]: make sure check the python version and quite the program if the version is lower than the minimum working version.

File: etc/setup.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    export PYCURL_SSL_LIBRARY=nss
+    pip install --compile --install-option="--with-nss" --no-cache-dir pycurl
+fi
+
+source ~/etc/var.sh
+export PATH=$PythonFolder:$PATH
+export PYTHONPATH=$PythonFolder
+export PATH=$HOME/.local/bin:$PATH
+export DBS3_CLIENT=$SitePackageFolder/DbsClient/
+export LD_LIBRARY_PATH=$(which curl)
```
Comment (yuyiguo) [inline etc/setup.sh:22]: why set LD_LIBRARY_PATH=$(which curl)?

File: etc/setup.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+# test
+if [ "$1" = "test" ];
+then
+    export DBS_WRITER_URL=https://cmsweb-testbed.cern.ch/dbs/int/global/DBSWriter/
+    export DBS_READER_URL=https://cmsweb-prod.cern.ch/dbs/prod/global/DBSReader/
+fi
+
+
+echo $DBS3_CLIENT
```
Comment (yuyiguo) [inline etc/setup.sh:34]: add 
echo $DBS_WRITER_URL
echo $DBS_READER_URL

File: setup.sh [hunk 5]
```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
```
Comment (yuyiguo) [inline setup.sh:1]: How this file different from etc/setup.sh?

File: tests/dbsclient_t/unittests/DBSClientBlockWriter_t.py [hunk 6]
```diff
@@ -6,8 +6,8 @@
 import time
 import uuid
 import unittest
-from dbs.apis.dbsClient import *
-from dbs.exceptions.dbsClientException import dbsClientException
+from dbsClient.apis.dbsClient import *
+from dbsClient.exceptions.dbsClientException import dbsClientException
 from RestClient.ErrorHandling.RestClientExceptions import HTTPError
 from ctypes import *
```
Comment (yuyiguo) [inline tests/dbsclient_t/unittests/DBSClientBlockWriter_t.py:12]: why not py3 json?

**Review by yuyiguo — CHANGES_REQUESTED** [review]
@Neilstid 

Please fix the comments and resubmit a PR.

File: tests/dbsclient_t/unittests/DBSClientBlockWriter_t.py (hunk 6 above)
Comment (Neilstid) [reply tests/dbsclient_t/unittests/DBSClientBlockWriter_t.py:12]: json is the integrated python3 library to read/write/... json

**Comment by Neilstid** [comment]
Change in the latest pull request taking care of all comment

**Pull request closed without merging**